zeroeq/http/helpers.h

```diff
@@ -0,0 +1,27 @@
+
+/* Copyright (c) 2017, Human Brain Project
+ *                     Stefan.Eilemann@epfl.ch
+ */
+
+#ifndef ZEROEQ_HTTP_HELPERS_H
+#define ZEROEQ_HTTP_HELPERS_H
+
+#include <future>
+
+namespace zeroeq
+{
+namespace http
+{
+
+/** @return ready future wrapping the value passed. */
+template<typename T>
+std::future<T> make_ready_future( const T value )
+{
+    std::promise<T> promise;
+    promise.set_value( value );
+    return promise.get_future();
+}
```

Comment:
Yes, it is missing from the standard. It could go to this future Extra project above ZeroEQ at some point, but now I see two solutions:
- make this header private like you had it and let users figure out how to make a ready Response.
- new alternative: keep the header public but change the function to:
std::future<Response> make_ready_response(...);
forwarding the arguments to the Response constructor, as that is the only use case for this helper. I think this would be quite nice to use!